### ISSUE TITLE
[BUG] Grouped Radiator Valves Do Not Show

### ISSUE THREAD

**Issue by AstersikRoss** [Pyhass/Hive-Custom-Component#148]
**Describe the bug**
Hive Radiator valves that are in a group do not show as individual valves or as a group. This means there is no way to control them in Home Assistant.

**To Reproduce**
Steps to reproduce the behaviour:

1) Group two or more valves in the Hive app
2) Sign into Hive via the Home Assistant Integration
3) View devices, only single valves are listed.

**Expected behaviour**
We would expect to see either the group of valves under one control or a group and individual valves

**Comment by MPSwitch** [comment]
Unfortunately doesn't look like this is active, having the same issue :(

**Comment by cpether** [comment]
I've created a PR to fix this, all you have to do is bump the version of pyhiveapi that it's using. Not sure if anyone is going to merge it in, but you can update it yourself in HA if you want to fix the issue.

https://github.com/Pyhass/Hive-Custom-Component/pull/153

**Comment by MPSwitch** [comment]
Sorry to sound silly, if this isn’t being monitored or updated to implement the new change is it a case of just changing the files you’ve changed in the PR?

**Comment by cpether** [comment]
Yes, i just used file editor in HA and changed the files then did a restart and it all worked. Or you can copy the files from my forked repo if you want: https://github.com/cpether/Hive-Custom-Component

**Comment by MPSwitch** [comment]
Cheers @cpether I have modified the file, my TRVs now show individually, i.e. Living Room Front and Living Room Back, and the controls for them appear on the first TRV in the group in the Hive app, i.e. no controls on Living Room Back but all the usual controls on Living Room Front. This looks to be just the solution! Many thanks, just read your forks documentation page, would it be possible to reset up my Hive integration with your fork specifically and have this all working? Excuse my limited knowledge of git, fairly new to all this.
